play-ahc-ws-standalone/src/test/scala/play/api/libs/ws/ahc/AhcWSRequestSpec.scala

```diff
@@ -108,6 +107,47 @@ class AhcWSRequestSpec extends Specification with Mockito with AfterAll with Def
 
     }
 
+    "with unencoded values" in {
```

Comment:
Some more tests would be good here, especially showing parameter names that are encoded.

Reply:
Added.